### ISSUE TITLE
Reformat and recomment code for mkdocstring

### ISSUE THREAD

**Issue by christopolise** [NET-BYU/sss#39]
`mkdocstring` is a great plugin for `mkdocs` which analyzes comments and generates documentation for the code. As can be seen in 1ae6fb0 the comments need to be modified so that every function/class conforms to the following format:

```python
def example_def(val1, val2, opt1):
    """
    Brief explanation of what the function does

    Args:
        val1 (int): Description of this var
        val2 (str): Description of this var
        opt1 (bool): Description of this option

    Returns:
        (list): Returns a list as a result
    """

    # Function code here # 
    ...
```

I am sure this will only need to be done for the code that acts as an API (i.e. display objects, controllers, broadcasters, etc.) but if we could all just take a sec and reformat that this, it will make the documentation much more usable. 

Thanks!

**Comment by christopolise** [comment]
Also it turns out we could do the following type of commenting as well if putting the type in parentheses is annoying:

```python
def example_def(val1: int, val2: str, opt1: bool) -> list:
    """
    Brief explanation of what the function does

    Args:
        val1: Description of this var
        val2: Description of this var
        opt1: Description of this option

    Returns:
        Returns a list as a result
    """

    # Function code here # 
    ...
```

**Comment by christopolise** [comment]
Resolved in #68
